cli/remotebazel/remotebazel.go

```diff
@@ -634,7 +644,8 @@ func Run(ctx context.Context, opts RunOpts, repoConfig *RepoConfig) (int, error)
 
 	req := &rnpb.RunRequest{
 		GitRepo: &rnpb.RunRequest_GitRepo{
-			RepoUrl: repoConfig.URL,
+			RepoUrl:     repoConfig.URL,
+			AccessToken: os.Getenv("GITHUB_TOKEN"),
```

Comment:
validate that the URL starts with `https://github.com/` before passing the token here?

Reply:
Unfortunately url.Parse can't parse `github` urls correctly (like `git@github.com:buildbuddy-io/buildbuddy.git`). Just used a simple string check